### DISCUSSION TITLE
404  File not found  The site configured at this address does not contain the requested file.  If this is your site, make sure that the filename case matches the URL. For root URLs (like http://example.com/) you must provide an index.html file.

### DISCUSSION THREAD

**Discussion by davidccarter** [community/community#57355] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Can someone please help me understand why my code isn't working to be displayed on my website (davidccarter.github.io). I keep getting this error message about the html file.

https://github.com/davidccarter/davidccarter.github.io

**Comment by mystieneko** [comment, marked as answer]
it's not named properly (you need to name it index.**html**, not index.**hmtl**)

**Reply by davidccarter** [reply]
Oh my goodness I'm so dumb. Thank you so much idk I how didn't see that
